downloads/index.md

```diff
@@ -1,6 +1,53 @@
 ---
 title: "Downloads"
 mirror: "https://github.com/wxWidgets/wxWidgets/releases/download"
+source_archives:
+  - description: "Windows ZIP"
+    prefix: "wxWidgets-"
+    postfix: ".zip"
+  - description: "Windows 7Z"
```

Comment:
This is _really_ nitpicking, but I think `7z` is much more usual, i.e. "z" shouldn't be capitalized.